modules/statics/src/coins.ts

```diff
@@ -7150,7 +7160,7 @@ export const coins = CoinMap.fromCoins([
   ofcerc20('5f936db2-6014-4afa-b997-4e7c6b34e814', 'ofcpro', 'Propy', 18, UnderlyingAsset.PRO),
   ofcerc20('65055519-d950-466a-b713-f88d9b1bde7c', 'ofcpundix', 'Pundi X2', 18, UnderlyingAsset.PUNDIX),
   ofcerc20('fb602049-f477-43ea-b938-95c2484eda31', 'ofcray', 'Raydium', 6, UnderlyingAsset.RAY),
-  ofcerc20('1c3eec6c-c1be-4ca9-b044-6e4379be0287', 'ofcreef', 'REEF', 18, UnderlyingAsset.RAY),
+  ofcerc20('1c3eec6c-c1be-4ca9-b044-6e4379be0287', 'ofcreef', 'REEF', 18, UnderlyingAsset.REEF),
```

Comment:
Breaking Change. Needs a breaking change comment in the commit footer.

Reply:
Added comment